loadtest/control/utils.go

```diff
@@ -276,31 +282,54 @@ func AttachFilesToPost(u user.User, post *model.Post) error {
 // AttachFilesToDraft uploads at least one file on behalf of the user, attaching
 // all uploaded files to the draft.
 func AttachFilesToDraft(u user.User, draft *model.Draft) error {
-	fileIDs, err := _attachFilesToObj(u, draft.ChannelId)
+	fileIDs, err := _attachFilesToObj(u, draft.ChannelId, rand.Intn(4))
 	if err != nil {
 		return err
 	}
 	draft.FileIds = fileIDs
 	return nil
 }
 
-func _attachFilesToObj(u user.User, channelID string) ([]string, error) {
+func AttachFileToBookmark(u user.User, bookmark *model.ChannelBookmark) error {
+	fileIDs, err := _attachFilesToObj(u, bookmark.ChannelId, 1)
+	if err != nil {
+		return err
+	}
+
+	bookmark.FileId = fileIDs[0]
+	return nil
+}
+
+func _attachFilesToObj(u user.User, channelID string, qtyToUpload int) ([]string, error) {
```

Comment:
A bit of a nit (it took me a while to understand what was happening here): Can we change the logic here a bit? I'm thinking of renaming `qtyToUpload` to something like `maxToUpload`, and let the callers pass an explicit number. Then, in the body of this function, do a `rand.Intn(maxToUpload)` and keep the maximum between that number and 1. Then, do the upload and counting.